code/modules/power/supermatter/supermatter.dm

```diff
@@ -676,7 +676,14 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			var/atom/movable/pulled_object = P
 			if(ishuman(P))
 				var/mob/living/carbon/human/H = P
-				H.apply_effect(40, EFFECT_KNOCKDOWN, 0)
+				if(istype(H.shoes, /obj/item/clothing/shoes/magboots)) //yogs start
+					var/obj/item/clothing/shoes/magboots/MB = H.shoes
+					if(MB.magpulse)
+						to_chat(H, "<span class='warning'>Your \the [MB] keep you standing up against [src]'s pull!</span>")
```

Comment:
it should never ever ever have the, so adding the \the is dumb

Reply:
"Your the magboots"?

Reply:
exactly

Reply:
The the is there whether or not I remove the \the, guess I forgot to remove this